### PR TITLE
NOW YOU GET TO BE A BIG SHOT

### PR DESCRIPTION
**Pull request by drexample**
## About The Pull Request

THIS [EXCLUSIVE] PULL REQUEST IS SO GOOD I'M GOING TO [hyperlink blocked] MYSELF! 
PROVIDE YOUR [First pet's name] AND [3 numbers on the back] TO RECEIVE FREE KROMERS AND A [COMMEMORATIVE MASK] FOR JUST [THREE EASY PAYMENTS OF 9.95]!

## Why It's Good For The Game

ARE YOU [DEAF]? ARE YOU [STUPID]? ARE YOU NOT HOLDING ONTO YOUR [RAPIDLY SHRINKING] LIFE?
THIS [COMMEMORATIVE MASK] WILL MAKE YOU [BETTER], WILL MAKE YOU [STRONGER], WILL MAKE YOU [HOT MILFS IN YOUR AREA]!

## Why It's Good For The Game

Sprite by Infrared Baron
image

## Changelog
:cl:
add: Adds a new wearable oddity - A Big Shot Mask. It will greatly boost your stats and provide strong protection (for a mask, at least), but at a steep price! (doubles as a gas mask too!)
/:cl:

### CODE REVIEW

**Comment by Benblu** [comment]
do this

**Review by mattroks101 — APPROVED** [review]
Code looks good.

**Comment by zombiemimic** [comment]
https://youtu.be/zhF8AgBR8hM

File: code/datums/perks/oddity.dm [hunk 1]
```diff
@@ -284,3 +284,226 @@
...
+				word = "\[[pick(list_of_great_deals)]\] [word]"
+		else
+			if(prob(75))
+				word = "[word] \[[pick(list_of_great_deals)]\]"
+			else
+				word = "  [word]  "
+
+		split_phrase[index] = word
+
+	return sanitize(jointext(split_phrase," "))
```
Comment (Iglpigl) [inline code/datums/perks/oddity.dm:509]: No newline.

File: code/game/objects/items/oddities.dm [hunk 2]
```diff
@@ -568,3 +568,29 @@
...
+	price_tag = 1997
+
+	spawn_tags = SPAWN_TAG_ODDITY
+
+/obj/item/clothing/mask/gas/big_shot/equipped(mob/living/carbon/human/user, slot)
+	..()
+	if(slot == slot_wear_mask)
+		user.stats.addPerk(/datum/perk/big_shot)
+		var/datum/perk/big_shot/perk = user.stats.getPerk(PERK_BIG_SHOT)
+		perk.my_mask = src
```
Comment (Iglpigl) [inline code/game/objects/items/oddities.dm:596]: No newline.

File: code/datums/perks/oddity.dm [hunk 3]
```diff
@@ -284,3 +284,226 @@
...
+		"Storing up for the winter!",
+		"On A Friday Night?",
+		"For A LimiTed Time Only!",
+		"TOP 10 HANSA CELEBRITIES!",
+		"Predstraza Gun Schematics!",
+		"Royalty Free Tapes!",
+		"Big",
+		"BIG",
+		"BIG! BIG BIG!",
+		"TELEKRYSTALS CHEAP",
```
Comment (SirRichardFrancis) [inline code/datums/perks/oddity.dm:329]:
```suggestion
		"TELECRYSTALS CHEAP",
```

File: code/datums/perks/oddity.dm [hunk 4]
```diff
@@ -284,3 +284,226 @@
...
+		"[pick(1, 5, 8, 1997)] KROMER CHEAP!",
+		"Burning acid",
+		"A Ride around the Colony on Our Specil Kestrel",
```
Comment (SirRichardFrancis) [inline code/datums/perks/oddity.dm:332]:
```suggestion
		"A Ride around the Colony on Our Special Kestrel",
```

File: code/datums/perks/oddity.dm [hunk 5]
```diff
@@ -284,3 +284,226 @@
...
+		"Oxycodone without prescription!",
+		"Replica watches with custom inscriptions!",
+		"is your old Earth coin worth 5000 credits?",
+		"maps of deepmaint download for free",
+		"proceed",
+		"PROCEED",
+		"one weird tip discovered by a mom",
+		"she looks half her age!",
+		"whoa nelly",
+		"it;s yours my friend",
```
Comment (SirRichardFrancis) [inline code/datums/perks/oddity.dm:378]:
```suggestion
		"it's yours my friend",
```

File: code/datums/perks/oddity.dm (hunk 3 above)
Comment (drexample) [reply code/datums/perks/oddity.dm:329]: All misspellings intentional

**Comment by TheObserver-sys** [comment]
There are things that are based, and then there are things that go beyond, that make me feel like a [BIG SHOT!!!!] on a [Ride Around Maintenance]

**Comment by Alliostra** [comment]
Is it april already?

**Review by TheShown911 — COMMENTED** [review]
Is this sans from undertale?

**Pull request merged**